packages/ui5-tooling-transpile/lib/task.js

```diff
@@ -294,11 +294,15 @@ module.exports = async function ({ workspace /*, dependencies*/, taskUtil, optio
 							return `/// <reference path=".${
 								/\.d\.ts$/.test(dtsFile) ? dtsFile : dtsFile.replace(/\.ts$/, ".d.ts")
 							}"/>`;
-						})
-						.join("\n");
+						});
+					indexDtsContent.unshift(
+						`// Generated with TypeScript ${ts.version || "unknown"} / ${
+							rootProject?.framework?.name || "UI5"
+						} ${rootProject?.framework?.version || "unknown"}`
```

Comment:
erm... I think the version of the types might be more important and may differ. Or both?

Reply:
Done!

![image](https://github.com/ui5-community/ui5-ecosystem-showcase/assets/1589419/f0d0bddb-743b-4ec4-b51a-ca5c8f12517f)